website/docs/01 - Introduction/01 - What is MVOM.md

```diff
@@ -9,15 +9,15 @@ MVOM (**M**ulti**V**alue **O**bject **M**apper) is a library which provides the
 
 ## Requirements
 
-:::warning
+:::danger
 The creators of MVOM work with Unidata exclusively. This has not been tested on Universe. If you would like to help us test and ensure everything works on Universe, please [log an issue](https://github.com/STORIS/mvom/issues/new) and let us know.
 :::
 
 - MultiValue Database Server
   - Unidata (version 8.2.1 or higher)
   - Universe (see warning above)
 - MultiValue Integration Server (version 1.3.0 or higher)
-- Node.js (version 16.0.0 or higher)
+- Node.js (version 18.0.0 or higher)
```

Comment:
This part gets a bit tricky. Generally speaking, we do not want to publish a new docs release until there is a new release published. That's because the docs would then be ahead of the release and there's an unknown time between now and when the release will be published.

The way we've generally handled this in the past is to create a working PR that has the changes necessary for the future release. That PR gets updated as docs need to change, and then immediately prior to publishing a release the docs PR is merged.

https://github.com/STORIS/mvom/pull/469 is an example of that.

So basically the changes you're making here to support node 18 and the new docusaurus are good. But any doc changes related to the node 18 update should go into a separate PR that will remain pending until a new release is made.

Reply:
Fixed.

Reply:
Please go ahead and get the separate docs PR created. That will be a living PR until the time a new release is published.

Reply:
Docs PR entered - [#621](https://github.com/STORIS/mvom/pull/621)